cmd/publishing-bot/main.go

```diff
@@ -53,7 +53,7 @@ func main() {
 	repoName := flag.String("source-repo", "", "the name of the source repository (eg. kubernetes)")
 	repoOrg := flag.String("source-org", "", "the name of the source repository organization, (eg. kubernetes)")
 	targetOrg := flag.String("target-org", "", `the target organization to publish into (e.g. "k8s-publishing-bot")`)
-	basePublishScriptPath := flag.String("base-publish-script-path", "./publish_scripts", `the base path in source repo where bot will look for publishing scripts`)
+	basePublishScriptPath := flag.String("base-publish-script-path", "/publish_scripts", `the base path in source repo where bot will look for publishing scripts`)
```

Comment:
i think this was meant to be a relative path inside the repo

Reply:
It's relative to the current working dir. I think before this PR it was coincidently the root of the container. Will check. I think we have to restore the current working dir correctly. Then the `./` is fine.

Reply:
fixed